src/mono/wasi/Wasi.Build.Tests/Wasi.Build.Tests.csproj

```diff
@@ -17,7 +17,7 @@
     <InstallWorkloadForTesting>true</InstallWorkloadForTesting>
 
     <!-- don't run any wasm build steps -->
-    <IsBrowserWasmProject>false</IsBrowserWasmProject>
+    <IsWasiProject>false</IsWasiProject>
```

Comment:
this looks odd

Reply:
This WBT project is not a *wasi* project, but because of the way it is built (arch=wasm, os=wasi) the wasi target files will get imported. Setting this property ensures that the wasi targets don't run, as they wouldn't be of any use.